survey.tex

```diff
@@ -2,6 +2,6 @@
 % Responsible : Zeljko
 \section{Survey Cadence}
 
-In this section we discuss the implications of any \es program on the  baseline cadence and overall survey strategy. 
-This section should reference the three early science scenarios presented in \ref{ssec:scenarios} and explain the implications on the survey cadence for each strategy. 
+Early Science observations must align with main survey and ultimate long-term science goals. 
```

Comment:
Early Science observations must align with main survey and ultimate long-term science goals. ->

Early Science observations should align as closely as possible with main survey and ultimate long-term science goals.